workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/DefaultApplyLatestTransformSystem.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Generated.Improbable.Transform;
+using Improbable.Gdk.Core;
+using Unity.Collections;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.Experimental.PlayerLoop;
+
+namespace Improbable.Gdk.TransformSynchronization
+{
+    [DisableAutoCreation]
+    [UpdateInGroup(typeof(FixedUpdate))]
+    public class DefaultApplyLatestTransformSystem : ComponentSystem
+    {
+        private struct RigidbodyData
+        {
+            [ReadOnly] public readonly int Length;
```

Comment:
These never need [ReadOnly]

Reply:
I thought we were doing this stylistically but sure. I didn't like this being here anyway.